src/Cardano/HttpBridge/Client.hs

```diff
@@ -0,0 +1,51 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE RankNTypes #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
This is _acceptable_ in testing to define Arbitrary instances (and subject to debate, probably hedgehog would be a better choice at some point), but let's not introduce any orphan instances in our source code.

Reply:
Agreed. I've removed them.